includes/functions/functions_email.php

```diff
@@ -385,24 +389,24 @@ function zen_mail($to_name, $to_address, $email_subject, $email_text, $from_emai
             if (isset($newAttachmentsList) && is_array($newAttachmentsList)) {
                 $attachments_list = $newAttachmentsList;
             }
-            if (defined('EMAIL_ATTACHMENTS_ENABLED') && EMAIL_ATTACHMENTS_ENABLED && is_array($attachments_list) && sizeof($attachments_list) > 0) {
+            if (defined('EMAIL_ATTACHMENTS_ENABLED') && EMAIL_ATTACHMENTS_ENABLED && is_array($attachments_list) && $attachments_list !== []) {
```

Comment:
should we use count on the last condition?

Reply:
Nope, since it's known that the variable is an array, checking the variable specifically for an empty array is more performant.